lib/internal/Magento/Framework/Data/Collection.php

```diff
@@ -492,6 +492,7 @@ public function clear()
     {
         $this->_setIsLoaded(false);
         $this->_items = [];
+        $this->_totalRecords = null;
```

Comment:
I don't see that private method removal. Other than that LGTM.

Reply:
Most likely, we don't need to remove this private method (`invalidateSize`), because it's being used in `\Magento\Eav\Model\Entity\Collection\AbstractCollection::addAttributeToFilter`.

Reply:
Surely it's used, but there is no reason for its existence. Simply put this method body in place where it is called.

Reply:
@orlangur this method (invalidateSize) is now in place where it is called, within \Magento\Eav\Model\Entity\Collection\AbstractCollection class

Reply:
@sergeynezbritskiy I'm asking you to get rid of that private method, how it was in original PR. Just do the same assignment instead of private method call and remove the method.

Reply:
@orlangur fixed